scripts/sign_encrypt.py

```diff
@@ -54,8 +58,8 @@ def get_args(logger):
         ' image file from\n' +
         '                 TA raw image and its signature. Takes' +
         ' arguments\n' +
-        '                 --uuid, --in, --key, --enc-key (optional), --out,' +
-        ' --algo (optional) and --sig.\n\n' +
+        '                 --uuid, --in, --key, --out, --enc-key (optional),' +
+        ' --enc-key-type (optional), --algo (optional) and --sig.\n\n' +
```

Comment:
Above `digest` command requires similar changes as well.

Reply:
I didn't see where the incorrect `,` placement is for this one but feel free to point it out.

Reply:
Actually I meant `digest` command requires `--enc-key-type (optional)` argument as well.

Reply:
done